JavaScriptSPA/index.html

```diff
@@ -3,32 +3,53 @@
 <head>
     <title>Quickstart for MSAL JS</title>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js"></script>
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/0.2.4/js/msal.js"></script>
+    <script src="/msal-0.9.0.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="/style.css">
 </head>
 
 <body>
-<h4 id="WelcomeMessage"></h4>
-<button id="SignIn" onclick="signIn()">Sign In</button>
-<br/><br/>
-<pre id="json"></pre>
-
+    <div class="container">
+        <div class="leftContainer">
+            <p id="WelcomeMessage">Welcome to the Microsoft Authentication Library For Javascript Quickstart</p>
+            <button id="SignIn" onclick="signIn()">Sign In</button>
+        </div>
+        <div class="rightContainer">
+            <pre id="json"></pre>
+        </div>
+    </div>
 <script>
     var applicationConfig = {
-        clientID: 'Enter_the_Application_Id_here', //This is your client ID
-        authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here",  //Default authority value is https://login.microsoftonline.com/common
-        graphScopes: ["user.read"],
-        graphEndpoint: "https://graph.microsoft.com/v1.0/me"
+        clientID: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
+            authority: "https://login.microsoftonline.com/common",
+            graphScopes: ["user.read", "Mail.Send"],
+            graphEndpoint: "https://graph.microsoft.com/v1.0/me",
+            loginType: 'POPUP'
+    };
+
+    var config = {
+        auth: {
+            clientId: applicationConfig.clientID,
+            authority: applicationConfig.authority,
+            validateAuthority: true
```

Comment:
validateAuthority is by default true. Can we remove the explicitly setting if not needed again? This will be one extra option to explain in the quickstart.